boudicca.base/publisher-event-html/src/main/resources/static/js/main.js

```diff
@@ -10,6 +10,34 @@ document.addEventListener("DOMContentLoaded", () => {
   const categorySelect = document.getElementById("category");
   const categoryFieldSets = document.querySelectorAll("[data-category-wanted]");
   const searchInput = document.querySelector("input.search-input");
+  const modal = document.getElementById("modal");
+  const modalContent = modal.querySelector("#modal-content");
```

Comment:
why do you use the querySelector method with ids instead of the normal getElementById?

Reply:
I think there is sth with async code that I don't know how to handle, when I access the modal-content by document. getElementById, I'm not able to fill it with the content later. however, I changed the selector for the close-button as was requested.